README.md

```diff
@@ -0,0 +1,57 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> This is just a simple project under the supervision of the Microverse platform that tests how descriptive our repo instruction (pull requests description, readme file), also Linters were installed in the project to check for internal code errors and make the code as clean as possible.
+
+# Project Name
+
+> Portfolio: setup and mobile first
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- You must have a code editor installed (VS code or Atom).
+
+### Setup
+
+- git init
+- git add .
+- git commit -m "first commit"
+- git branch -M
+- git clone
+
+## Authors
+
+:bust_in_silhouette: **Author1**
+
+- GitHub: [@githubhandle](https://github.com/AhmedMohsen600)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ahmed-mohsen-bb75b2194/)
+
+## :handshake: Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/AhmedMohsen600/Hello-Microverse/issues).
+
+## Show your support
+
+Give a :star:️ if you like this project!
+
+## Acknowledgments
+
+- Microverse documentation.
+
+## :memo: License
+
+You can freely use this project for ethical use and educational purposes.
+GitHubGitHub
+AhmedMohsen600 - Overview
+AhmedMohsen600 has 34 repositories available. Follow their code on GitHub.
```

Comment:
- [ ] Kindly add a license file in your project using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1:
- [ ] After adding the license file in your project kindly add the link of your license file here :+1: